module/purpose/FieldPlayer/src/FieldPlayer.cpp

```diff
@@ -90,27 +92,50 @@ namespace module::purpose {
                     return;
                 }
 
+                // Make an ignore list with the goalie, if they exist
+                std::vector<unsigned int> ignore_ids{};
+                // Add inactive robots to the ignore list
+                for (const auto& robot : (robots ? robots->robots : std::vector<message::localisation::Robot>{})) {
```

Comment:
imo it's less clunky as
```
if (robots)
    for (const auto& robot: robots->robots) {...}
```
rather than iterating over an empty vector

Reply:
just noticed this is already done on line 224 anyway, so good for consistency too